Resources/config/doctrine/model/Option.orm.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <doctrine-mapping xmlns="http://doctrine-project.org/schemas/orm/doctrine-mapping" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://doctrine-project.org/schemas/orm/doctrine-mapping http://doctrine-project.org/schemas/orm/doctrine-mapping.xsd">
     <mapped-superclass name="Ekino\WordpressBundle\Model\Option">
-        <field name="name" type="string" length="64" column="option_name" unique="true">
```

Comment:
Using `unique` this way does not allow to customize index name so I've moved their definitions to `Resources/config/doctrine/Option.orm.xml` and `Resources/config/doctrine/Term.orm.xml`. :panda_face: